ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/fragmentisers/geospatial/connected/relations/GeospatialRelationsAttributer.java

```diff
@@ -11,12 +11,12 @@
 
 import java.util.List;
 
-import static be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.constants.GeospatialConstants.GEOSPARQL_AS_WKT;
-import static be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.constants.GeospatialConstants.TREE_GEOSPATIALLY_CONTAINS_RELATION;
+import static be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.constants.GeospatialConstants.*;
```

Comment:
Why not only import what you need?

Reply:
According to Intellij the rule is, once you need to import three -> import via .*